cost/flexera/cco/cheaper_regions/cheaper_regions.pt

```diff
@@ -46,19 +47,29 @@ credentials "auth_flexera" do
 end
 
 ###############################################################################
-# Resources
+# Datasources & Scripts
```

Comment:
The purpose of naming this section "Datasources & Scripts" is that datasources and scripts should be in one section of the policy rather than two sections. Please delete the "Scripts" section and place each script underneath the datasource that is calling it.